### ISSUE TITLE
while use pm2 run the project，the log4js log is not work

### ISSUE THREAD

**Issue by Sailor20** [nomiddlename/log4js-node#510]
while use pm2 run the project，the log4js log is not work, what should i do?

**Comment by nomiddlename** [comment]
It's always a good idea to check the existing issues on a project before creating a new issue. PM2 support is covered by this issue: #265 - and this PR #509 - once that pull request is merged (which will be soon), logging will work with PM2. I'm going to close this issue, because #265 already covers this. Thanks.
